src/pages/OurSupporters.tsx

```diff
@@ -170,76 +167,26 @@ export default function OurSupporters() {
     },
   ]
 
-  /*   const cardSliderData = [
-      {
-        route: Card1VH,
-        name: "Annelisse Torrez Daza",
-        textLabel: "“Having suffered through COVID myself along with all of my family in a underprivileged country, I knew instantly that I could make a difference and prevent others from facing my same fate by joining Virufy.”",
-        country: "Bolivia",
-      },
-      {
-        route: Card2VH,
-        name: "Sebastian Pedraza",
-        textLabel: "“Virufy unites the power of youth and technology to transform the world. We hope that this solution will reach the whole world and that it will become a support for doctors, especially in low-income countries.”",
-        country: "Columbia",
-      },
-    ] */
-
   return (
-    <>
-      <TextinsImg
-        sizeImg="auto"
+    <div className='h-full bg-[#4064AD]'>
+
+      {/* Globe background img */}
+      <TextinsImg2
+        sizeImg=""
         borderImg="none"
         src={ImageinsText}
-        altImg="Imagen cabezera"
-        Text={titleImage}
+        altImg="background img"
```

Comment:
I recently learned that if the image is decorative, such as this background image, set `alt` to an empty string so assistive tech can skip it.
For more info: https://www.w3.org/WAI/tutorials/images/decision-tree/

Reply:
what is assistive tech?

Reply:
technology like a screen reader